docs/helm/configure-namespaced-helm-repos.md

```diff
@@ -0,0 +1,85 @@
+# Configure Namespace Scoped Chart Repositories using namespaced CRDs
+
+While the cluster-scoped CRD `HelmChartRepository` for Helm repository provides ability for admins to add Helm repositories as custom resources, regular users/developers need an admin's permission in order to add Helm repositories of their choice.
+
+To avoid permission elevation, use namespace scoped CRD `ProjectHelmChartRepository`.
+
+## Basic structure of `ProjectHelmChartRepository`
+
+```yaml
+apiVersion: helm.openshift.io/v1beta1
+kind: ProjectHelmChartRepository
+metadata:
+  name: my-enterprise-chart-repo
+spec:
+  url: https://my.chart-repo.org/stable
+
+  # optional name that might be used by console
+  name: myChartRepo
+
+  # optional and only needed for UI purposes
+  description: my private chart repo
+
+  # required: chart repository URL
+  connectionConfig:
+    url: HELM_CHART_REPO_URL
+```
+
+## Getting Started with `ProjectHelmChartRepository`
+
+To add a new namespace-scoped Helm repository `stable` to a desired namespace `my-namespace`:
```

Comment:
Suggestion: To add a new namespace-scoped Helm repository, add a custom Helm Chart Repository CR, for example, `stable` CR to your `my-namespace` namespace: